pyramid_oereb/contrib/print_proxy/mapfish_print/mapfish_print.py

```diff
@@ -497,16 +497,17 @@ def convert_to_printable_extract(self, extract_dict, feature_geometry):
         extract_dict['RealEstate_RestrictionOnLandownership'] = restrictions
         # End one restriction entry per theme
 
-        for i, item in enumerate(extract_dict.get('Disclaimer', [])):
+        # Flatten the disclaimer dictionary
+        for item in extract_dict.get('Disclaimer', []):
             self._multilingual_text(item, 'Title')
             self._multilingual_text(item, 'Content')
 
-            if i == 0:
-                extract_dict['DisclaimerLandRegister'] = item
-                self._flatten_object(extract_dict, 'DisclaimerLandRegister')
-            if i == 1:
-                extract_dict['DisclaimerPollutedSites'] = item
-                self._flatten_object(extract_dict, 'DisclaimerPollutedSites')
+        # Separate the first item, so it can be placed in a different column in the template
+        disclaimer = extract_dict.get('Disclaimer', [])
+        if len(disclaimer) > 0:
+            extract_dict['DisclaimerLandRegister'] = disclaimer.pop(0)
```

Comment:
what about the other disclaimers, shall they be dropped ?

>> edit: I saw in mfp repo that disclaimers become a datasource, so this is fine, thanks
https://github.com/openoereb/pyramid_oereb_mfp/pull/88/files#diff-3c2b1d00d32412910451c36351e2094e2430d12d3cf966a8059a8123e88011a0R163